variables.tf

```diff
@@ -2,8 +2,60 @@
 # Input Variables
 ########################################################################################################################
 
-#variable "my_variable" {
-#  type        = string
-#  description = "A description of my variable"
-#  default     = "default_value"
-#}
+variable "resource_group_id" {
+  description = "The Id of an existing IBM Cloud resource group where the watsonx Assistant instance will be grouped."
+  type        = string
```

Comment:
Without a default value, this becomes a required variable. However it would not be required if user was passing `existing_assistant_instance` right? So I think it needs to have a default value of `null`, but add cross variable validation to ensure a value is passed for it if `existing_assistant_instance` is null.

Suggest to update variable description to ensure thats clear

Reply:
done